bugalink-backend/trips/urls.py

```diff
@@ -1,14 +1,17 @@
 from django.urls import include, path
 from rest_framework import routers
-
-from trips.views import TripRequestViewSet, TripViewSet
+from trips.views import TripRecommendationViewSet, TripRequestViewSet, TripViewSet
 
 router = routers.DefaultRouter()
 router.register(r"trips", TripViewSet)
 
 urlpatterns = [
     path("", include(router.urls)),
     path("trips/<int:id>/request/", TripRequestViewSet.as_view({"post": "create"})),
+    path(
+        "trips/<int:user_id>/recommendations",
```

Comment:
The endpoint was supposed to be `/recommendations` and the `user_id` would be deduced automatically from the authorization token. This endpoint estructure suggests that I could do `/trips/69/recommendations` which means two things:

* I'm referring to trip with ID 69 (which is not true, because that id belongs to a user)
* I'm obtaining another user's recommendations (which should not be possible)

This endpoint name raises another problem, which is that we should be adding a validation for checking recommendations that don't belong to our user. Both problems are solved if the endpoint is named just `/recommendations` as I wrote on the excel